Flowchart/Controllers/FlowchartController.cs

```diff
@@ -7,19 +7,26 @@
 using Microsoft.EntityFrameworkCore;
 using FlowchartCreator.Data;
 using FlowchartCreator.Models;
+using Microsoft.AspNetCore.Authorization;
 
 namespace FlowchartCreator.Controllers
 {
+    // DEBUG: Comment/Uncomment this to require authentication to access this controller's components.
+    [Authorize]
```

Comment:
Probably want to comment this out before pushing to the master branch. I might have missed it.